lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -133,13 +134,16 @@ def instructor_dashboard_2(request, course_id):  # lint-amnesty, pylint: disable
 
     sections = []
     if access['staff']:
-        sections.extend([
+        sections_content = [
             _section_course_info(course, access),
             _section_membership(course, access),
             _section_cohort_management(course, access),
-            _section_discussions_management(course, access),
```

Comment:
is this tab is only available for `user.is_staff` users?

Reply:
I think so. There is a check right above this at line 136. `if access['staff']`